client/helpers_test.go

```diff
@@ -662,136 +662,8 @@ func TestApplyTargetsChangeInvalidType(t *testing.T) {
 	assert.Error(t, err)
 }
 
-// A delegated role MUST NOT have both Paths and PathHashPrefixes defined.
-// These next 2 tests check that attempting to edit an existing role to
-// create an invalid role errors in both possible combinations.
-func TestApplyTargetsDelegationConflictPathsPrefixes(t *testing.T) {
```

Comment:
This test is no longer relevant as PathHasPrefixes have been removed and this tests that we error when adding hash prefixes to a role that already has paths.